app/assets/javascripts/admin/addon/services/site-setting-change-tracker.js

```diff
@@ -80,6 +83,49 @@ export default class SiteSettingChangeTracker extends Service {
     this.dirtySiteSettings.forEach((siteSetting) => siteSetting.cancel());
   }
 
+  refreshPage() {
+    document.documentElement.style.setProperty(
+      "--font-family",
+      this.siteSettings.base_font
+    );
+    document.documentElement.style.setProperty(
+      "--heading-font-family",
+      this.siteSettings.heading_font
+    );
+    document.documentElement.classList.remove("text-size-smaller");
+    document.documentElement.classList.remove("text-size-normal");
+    document.documentElement.classList.remove("text-size-larger");
+    document.documentElement.classList.remove("text-size-largest");
```

Comment:
Just wondering if we can just remove via a regex here.

Reply:
classList is not suporting regex but at least I could loop through constant you suggested earlier 💪